lib/main.js

```diff
@@ -17,7 +17,7 @@ Object.assign(exports, {
     // options.loadReason can be install/enable/startup/upgrade/downgrade
     PlacesProvider.links.init();
     SearchProvider.search.init();
-    options.telemetry = false;
+    options.telemetry = true;
```

Comment:
we need an additional pref for performance tracking. maybe we can call it 'profiling mode'?